symplyphysics/core/symbols/symbols.py

```diff
@@ -267,5 +267,22 @@ def clone_symbol(source: SymbolNew,
     return SymbolNew(display_symbol_new, source.dimension, display_latex=display_latex_new, **assumptions)
 
 
+def clone_function(
```

Comment:
The idea was to make a helper procedure that would convert a symbol into a function so that we would only need to define symbols and convert them to functions whenever needed. I'm guessing I interpreted the naming in a peculiar way, so maybe `clone_as_function` is better.

Reply:
Yes, naming is wrong. `clone_as_function` sounds good.